### PR TITLE
notification & email changes

### PR DESCRIPTION
**Pull request by ahsu1230**
want to...
1) fix notification link
2) remove picture from notifications
3) remove universal auth token from both emails & notifications

@andrewconner @yipingliao

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala [hunk 1]
```diff
@@ -20,17 +20,16 @@ class LibraryInviteEmailSender @Inject() (
...
-      (user, library, libOwner)
+      (library, libOwner)
     }
-    val libLink = s"""www.kifi.com${Library.formatLibraryPath(libraryOwner.username, libraryOwner.externalId, library.slug)}?auth=${library.universalLink}"""
+    val libLink = s"""www.kifi.com${Library.formatLibraryPath(libraryOwner.username, libraryOwner.externalId, library.slug)}"""
     val emailToSend = EmailToSend(
       fromName = Some(Left(fromUserId)),
       from = SystemEmailAddress.NOTIFICATIONS,
-      subject = s"${requestingUser.firstName} ${requestingUser.lastName} invited you to follow ${library.name}.",
+      subject = s"{@fullName(fromUserId)} invited you to follow ${library.name}!",
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala]: @joshm1 kinda like this?

**Comment by ahsu1230** [comment]
extends #11010

**Comment by andrewconner** [comment]
No picture?

**Comment by ahsu1230** [comment]
yeaa. I think I was having problems with it before, but now I think I know how to get the correct picture.
Also, right now the notification link only works if you stay on kifi.com. For some reason, clicking on it doesn't route correctly =/

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala [hunk 2]
```diff
@@ -20,17 +21,16 @@ class LibraryInviteEmailSender @Inject() (
...
-    val (requestingUser, library, libraryOwner) = db.readOnlyReplica { implicit session =>
-      val user = basicUserRepo.load(fromUserId)
+    val (library, libraryOwner) = db.readOnlyReplica { implicit session =>
       val library = libraryRepo.get(libraryId)
       val libOwner = basicUserRepo.load(library.ownerId)
-      (user, library, libOwner)
+      (library, libOwner)
     }
-    val libLink = s"""www.kifi.com${Library.formatLibraryPath(libraryOwner.username, libraryOwner.externalId, library.slug)}?auth=${library.universalLink}"""
+    val libLink = s"""www.kifi.com${Library.formatLibraryPath(libraryOwner.username, libraryOwner.externalId, library.slug)}"""
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/emails/LibraryInviteEmailSender.scala:29]: `https://`?

**Pull request merged**